news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,133 @@
+
+- Block reassurance: Now available on all pages (above or below the header and / or the footer) in addition to product pages and during the checkout. You can customize the color of the icon, add a description and a link. With this module you build the confidence of your visitors and thus improve your conversion rate.
+
+
+### Notable fixes
+
+More than [160 bugs](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aclosed+label%3Abug+milestone%3A1.7.7.0+-label%3ARegression+) have been fixed for the 1.7.7 (vs 110 for the 1.7.6.0), including [10 highly expected bug fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aclosed+label%3ATopwatchers+label%3Afixed+milestone%3A1.7.7.0), [5 front-office notable fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AFO+label%3Amust-have+label%3Afixed+milestone%3A1.7.7.0+-label%3Atopwatchers+) and [5 back-office notable fixes](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABO+label%3Amust-have+label%3Afixed+milestone%3A1.7.7.0+-label%3Atopwatchers+-label%3Aregression)
+
+
+### Notable technical improvements
+
+- **Updated PHP support.** Following [our announcement last year](https://build.prestashop.com/news/announcing-end-of-support-for-obsolete-php-versions/), PrestaShop 1.7.7 introduces support for PHP  7.3 while dropping support for PHP versions earlier than 7.1.3.
+
+- **Support for emojis.** All database tables are now encoded in utf8bm4 by default, enabling support for emojis everywhere 
+
+- **Updated jQuery to 3.4.1 everywhere.** Thanks to [jquery migrate](https://github.com/jquery/jquery-migrate), we have been able to upgrade jQuery to the latest version in Back Office as well as Classic & Core theme, without introducing backwards incompatible changes.
+- **Automatic form rendering.** Thanks to the newly updated Twig form theme, back office forms are now much easier to render. This allows extensions to [add form fields anywhere in the form](https://devdocs.prestashop.com/1.7/modules/sample_modules/grid-and-identifiable-object-form-hooks-usage/#adding-new-form-field-to-customer-form) without need to override the form’s template. (Note: this is being implemented progressively, not all forms support it at the time of writing)
+
+- **New CQRS Debugger in the Symfony debug bar.** With this new feature you can find out which commands and queries have been used to build the page you’re seeing. If you need to customize its behavior, you can [decorate or override it](https://devdocs.prestashop.com/1.7/modules/concepts/services/#override-an-existing-symfony-service) from a module.
+
+- **Improved tab management.** Tabs now support symfony routes (no longer need a fake legacy controller), automatic detection in modules, enable/disable
```

Comment:
not sure I understand the end of the sentence.
+ What is it useful ?

Reply:
@jolelievre

Reply:
When a module is disabled the Tab is now hidden (and when it is re-enabled it is visible again)

Reply:
All right, I added this info, let me know if it's clear enough @jolelievre